event/examples_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package event_test
+
+import (
+	"context"
+	"log"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/event"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// Event examples
+
+func ExampleCommandMonitor() {
+	startedCommands := make(map[int64]bson.Raw)
```

Comment:
Add a comment at the top of the function explaining what this monitor does. Also add a comment to say that this would have to be a concurrent map like `sync.Map` if the application makes multiple concurrent requests.

Reply:
done